### PR TITLE
Licensing-updates

### PR DESCRIPTION
**Pull request by amenne**
Updated licensing guidance based on new plans and tags

### CODE REVIEW

**Comment by amenne** [comment]
Hey @SLDam @pjlast @mike-r-mclaughlin - I raised this PR to clean up our licensing guidance and wrote a dedicated page on first time license creation. Can I get a review on this? Thanks!

File: content/departments/technical-success/ce/process/create-new-license.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+> [!NOTE] CEs should always consult with Sales before creating license keys for prospects (i.e., companies that have not yet officially become customers).
+
+> [!NOTE] Starting with Sourcegraph 5.1+, we require each customer instance to have a unique site_id and license key pair. Each instance should also have it's own subscription created. See [internal licensing FAQ](https://docs.google.com/document/d/1xzlkJd3HXGLzB67N7o-9T1s1YXhc1LeGDdJyKDyqfbI) for more details.
+
+> [!NOTE] The Subscription ID and Opportunity ID used below are **not** the same as the Subscription or Opportunity number. The correct data can be found in the Opportunity under Important Technical Details -> License IDs. It can also be pulled from the URL of the corresponding Subscription or Opportunity page.
+
+The CE should first create a Sourcegraph.com user account for the prospect/customer. Assuming there is not one already. The username of the account should have the following format `CompanyName-UniqueId`. This is format is described below. The email of the account should be left blank. Once that is available, follow the steps below.
```
Comment (pjlast) [inline content/departments/technical-success/ce/process/create-new-license.md]:
```suggestion
The prospect/customer requires a Sourcegraph.com user account. If an account does not already exist, the CE should create one first. The username of the account should have the following format `CompanyName-UniqueId`. This format is described below. The email of the account should be left blank. Once the account is available, follow the steps below.
```

File: content/departments/technical-success/ce/process/create-new-license.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Sign in to [site admin](license_keys.md#accessing-site-admin).
+1. Assuming this is the first time creating a license for this account you must first create a user by navigating to the [users page](https://sourcegraph.com/site-admin/users) and create new user account.
+   - You can also check for a user already exists by searching for a valid account (the uniqueID should match the Salesforce Account ID).
```
Comment (pjlast) [inline content/departments/technical-success/ce/process/create-new-license.md]:
```suggestion
   - You can also check if a user already exists by searching for a valid account (the uniqueID should match the Salesforce Account ID).
```

File: content/departments/technical-success/ce/process/create-new-license.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+1. Click **Generate license**.
+1. If the customer should have **Cody Access** once you generate an active license you are now able to toggle on `Access to hosted Cody Services`. Click Enable when the warning shows.
+   1. Everything should fill in for you; however if you need to modify the rate limit of chat requests per day or code completions per day you can do so by clicking on the pencil icon.
+1. Follow our [key sharing policy](license_keys.md#license-key-sharing-policy) for sharing this license key with your prospect or customer. You can link them to the following docs for instructions on where to add the key: [Updating your license key](https://sourcegraph.com/docs/admin/subscriptions#updating-your-license-key)
+1. Finally, a new customer, you must map the license key to the Salesforce instance for usage data tracking following [these directions](https://docs.google.com/document/d/12W85VTKLJg2Os74PWADxwOPfpMozB0mUm4Do6fN9dFs/edit?usp=sharing).
```
Comment (pjlast) [inline content/departments/technical-success/ce/process/create-new-license.md]:
```suggestion
1. Finally, if this is a new customer, you must map the license key to the Salesforce instance for usage data tracking following [these directions](https://docs.google.com/document/d/12W85VTKLJg2Os74PWADxwOPfpMozB0mUm4Do6fN9dFs/edit?usp=sharing).
```

File: content/departments/technical-success/ce/process/license_keys.md [hunk 4]
```diff
@@ -104,78 +286,10 @@ Within Sourcegraph we use 1Password for managing our credentials, including lice
...
-
-`plan:enterprise-0` represents our legacy enterprise pricing plan. `plan:enterprise-1` was launched in with [Sourcegraph 4.0](https://about.sourcegraph.com/pricing).
-
-These tags are supported but not currently saleable: - `branding`: Whether custom branding of this Sourcegraph instance has been purchased. - `backup-and-restore`: Whether builtin backup and restore on this Sourcegraph instance has been purchased.
+### Out of Contract License Extensions
 
-If no `plan:*` tag is supplied, the license will be treated as legacy enterprise tier which has unlimited access to all features.
+In select circumstances, such as a bridge extension being requested due to the renewal process not aligning with license expiration, Sales **must receive approval from the VP of Finance and VP of Sales to issue a licensing exception**. This approval is granted via request in #deal-desk. If an AE requests a deviation from their contract licensing terms, please validate that the necessary approvals have been granted _before_ making any changes to a customers' license key.
```
Comment (pjlast) [inline content/departments/technical-success/ce/process/license_keys.md]:
```suggestion
In select circumstances, such as a bridge extension being requested due to the renewal process not aligning with license expiration, Sales **must receive approval from the VP of Finance and VP of Sales to issue a licensing exception**. This approval is granted via request in #deal-desk. If an AE requests a deviation from their contract licensing terms, please validate that the necessary approvals have been granted _before_ making any changes to a customer's license key.
```

**Review by pjlast — APPROVED** [review]
Content looks good, just suggested some wording changes.

**Pull request merged**